services/licenses.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+
+function normalizeTemplate(template) {
+  const { title } = template;
+  return title.replace(/^Template:/, '');
+}
```

Comment:
I find this method weirdly named. From the name `normalizeTemplate` I would assume to get a template-object as input and a normalized template-object as output. But the output seems to be a string. So maybe `normalizedTemplateTitle` would be a better name?

Reply:
Good point! It did not really occur to me 🙈 I was only thinking about the **input** of the function, rather than the return value. But yeah, looking at it from that perspective, it makes totally sense to have the naming reflect that it's in the end only about the title.